### PR TITLE
SQ-27/Settings Part 4: licences ⚖️

### LINKED ISSUE #27: Screen: settings
Items:

 * [x] Log in/log out<sup>1</sup> (done in #177)
* [x] Enable/disable notifications (done in https://github.com/rock3r/squanchy/pull/136)
 * [x] Configure wifi<sup>1</sup>
  * [x] FOSS licences (done in #192)
 * [x] About the app:
    * [x] version info (done in https://github.com/rock3r/squanchy/pull/153)
    * [x] repo URL
    * [x] other?

Notes:
<sup>1</sup> not part of MVP, see login story

### PR DESCRIPTION
**Pull request by rock3r**
This PR closes #27 by implementing the last bit, the licences screen. As a side note I realised we don't need Android-Apt as since the Android Gradle Plugin 2.2 there is a built-in annotation processor, so I switched to that one and removed one dependency.

licences

TODO:
 * Edge cases _[BLOCKED waiting for design on signed off settings]_
 * Sub-screens:
    * ~FOSS licences~
    * ~About screen~
 * ~Use CoordinatorLayout in settings~
 * ~Photo border~
 * ~Obtain cover photo~ defo not worth it

### CODE REVIEW

**Comment by rock3r** [comment]
Don't be scared by the number of lines changed, a lot of it is just the licences' text :)

File: app/src/main/java/net/squanchy/about/licenses/LicensesActivity.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        setSupportActionBar(toolbar);
+        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
+        getSupportActionBar().setDisplayShowTitleEnabled(true);
+    }
+
+    @Override
+    protected void onStart() {
+        super.onStart();
+
+        licensesList.setAdapter(new LibrariesAdapter(this));
```
Comment (quiro91) [inline app/src/main/java/net/squanchy/about/licenses/LicensesActivity.java]: just asking: why are we creating the `LibrariesAdapter` every time the `Activity` is started? This is a fixed list which is never going to change at runtime so wouldn't it be better to set it in the `onCreate` ? Same for the `LayoutManager` and the `ItemDecorator`

Comment (rock3r) [reply app/src/main/java/net/squanchy/about/licenses/LicensesActivity.java]: Yep that's actually a bug, I created `onStart` by mistake and didn't notice. Will fix asap.

Comment (rock3r) [reply app/src/main/java/net/squanchy/about/licenses/LicensesActivity.java]: 4a98595

**Review by quiro91 — APPROVED** [review]
🎉

**Pull request merged**